pkg/mcs/tso/server/grpc_service.go

```diff
@@ -144,6 +144,13 @@ func (s *Service) FindGroupByKeyspaceID(
 		}, nil
 	}
 
+	if s.IsClosed() {
```

Comment:
It seems that validRequest has already check it?

Reply:
oh, sure